frontend/lib/src/components/shared/AlertContainer/AlertContainer.tsx

```diff
@@ -27,7 +27,7 @@ export enum Kind {
   WARNING = "warning",
 }
 
-function getNotificationKind(
+export function getNotificationKind(
```

Comment:
Does this need to be exported?

Reply:
oops I was playing with this function so I exported it. Will revert thanks.